extensions/opentelemetry/deployment/src/main/java/io/quarkus/opentelemetry/deployment/tracing/TracerProcessor.java

```diff
@@ -49,6 +50,11 @@ public class TracerProcessor {
     private static final DotName SPAN_PROCESSOR = DotName.createSimple(SpanProcessor.class.getName());
     private static final DotName TEXT_MAP_PROPAGATOR = DotName.createSimple(TextMapPropagator.class.getName());
 
+    @BuildStep
+    OpenTelemetryTracerEnabledBuildItem tracerEnabled() {
+        return new OpenTelemetryTracerEnabledBuildItem();
```

Comment:
I think we have a configuration that enables / disables the Tracer. Shouldn't we check the configuration before producing the build item?